### PR TITLE
P0 avatar

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153192044-a031f31d-a23c-45a6-b524-29f6b8a997bf.mp4

### CODE REVIEW

**Review by abhij1607 — APPROVED** [review]
Looks good

**Review by ZuberDunge — COMMENTED** [review]
you can remove extra spacings and use variables for colors

File: components/avatar.html/avatar.css [hunk 1]
```diff
@@ -0,0 +1,28 @@
+.avatar {
+    width: 50px;
+    height: 50px;
+    border-radius: 50%;
+  }
+
+  .size-large{
+    width: 70px;
```
Comment (purvasharmadev) [inline components/avatar.html/avatar.css]: use rems.

Comment (stutisk) [reply components/avatar.html/avatar.css]: made the changes , thanku

**Comment by stutisk** [comment]
changes made thanku

**Pull request merged**